.github/.wordlist.txt

```diff
@@ -841,6 +842,7 @@ MoveToSaturation
 MoveWithOnOff
 MPSL
 MRP
+MSR
```

Comment:
is MSR a word? I see it used as an acronym, however that is the only use. Could we just not use the acronym or use backticks to escape it from being considered a real word?